grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -98,6 +98,13 @@ class Location implements Comparable<Location>, java.io.Serializable {
             if (activities?.contains(ActivityCode.NONE.id) && activities?.size() > 1) {
                 return ['invalid.supportedActivities']
             }
+            // Don't allow to set REQUEST_APPROVAL activity if there are not any users that
+            // have Request Approver location role for this location
+            // TODO: Replace ROLE_APPROVER with the ROLE_REQUEST_APPROVER when OBPIH-5799 is done
+            if (activities?.contains(ActivityCode.REQUEST_APPROVAL.id)
```

Comment:
Was it meant to be a validator? I thought it was a message to inform the user that he has to do this **after** adding this supported activity to the location

Reply:
I apologize, I was confused because of many different versions that we were speaking about, but I've consulted with Katarzyna and she confirmed this could be just a message, not a validation.
I fixed that part.